JavaScript/Topics/13.DOM_in_javascript/README.md

```diff
@@ -0,0 +1,155 @@
+<br>
+It  returns the content present in the < body> tag. This property is used to view or change the  content present inside the < body> element and sets them with the new specified content.
+
+E.g. 
+```
+ var x = document.body; // Assigns body element to x 
+
+console.log(x.innerHTML); // Print text inside body element 
+
+//New content inside body can also be added 
+ x.innerHTML = "<h1>"
+                + "New Heading added "
+                + "</h1>" + "<br>" + "<h2>"
+                + "New Content Added"
+                + "</h2>";
+```
+
+### HTML | DOM forms Collection
+<br>
+The forms collection returns a collection of all < form> elements present in the document . Since there may be multiple forms in a page , the elements are sorted as they are present in the HTML source code . 
+
+E.g .
+ ```
+ //Assigns forms collection to x
+var x= document.forms ;
+ 
+ // Assigns numbers of form elements present  in the collection 
+var len= x.length ; 
+
+//To acsess any particular form element we can use indexing like arrays 
+// Assigns 1st form element
+var form1= x[0] ;
+
+//Or we can also use the below method 
+var form1 = x.item(0);
+
+ ```
+
+  The name attribute which specifies the name of an < input > element is used to reference form elements in a JavaScript, or to reference form data after a form is submitted.
+
+  E.g.
+ ```
+  
+ <form name="myForm" action="/action_page.php" method="get">
+ ...
+ </form>
+ //For the above HTML form element , We can access the form with name attribute as below
+
+  var x= document.forms["myForm"];
+  x.submit(); //Submits form
+ ```
+It is important to note that only form elements with a name attribute will have their values passed while submitting a form.
+
+<br>
+To access any particular element , DOM provides various method . Some of them are as follows :
+
+<br>
+
+<br>
+
+| Methods  | Use |
+| ------------ | ------------- |
+| document.getElementById(id) |  Find an element by element id |
+| document.getElementsByName(name) | Finds any element by its name attribute |
+| document.getElementsByTagName(name)  | Find elements by tag name  |
+| document.getElementsByClassName(name)  | 	Find elements by class name  |
+
+<br>
+<br>
+
+## Adding and Deleting Elements
+<br>
+DOM provides various methods to add and delete HTML elements via javascript . Some of them are as follows :
+<br>
+<br>
+
+| Method |	Description|
+|--- |----|
+|document.createElement(element) |	Create an HTML element |
+ | document.removeChild(element) |	Remove an HTML element|
+|document.appendChild(element)	|Add an HTML element |
+|document.replaceChild(new, old)	| Replace an HTML element |
+|document.write(text)	|Write into the HTML output stream |
+
+<br>
+
+This was a brief  note about DOM .Above methods are frequently used methods , apart from them DOM contains much more methods for manipulating HTML elements . To refer more about DOM , Please visit : [DOM](https://www.w3schools.com/js/js_htmldom_document.asp) 
```

Comment:
Add this line at the end
**Contributor**: [Manish Kumar Pandit](https://github.com/manishkumar-hub)
`**Contributor**: [Manish Kumar Pandit](https://github.com/manishkumar-hub)`